mimic/test/test_heat.py

```diff
@@ -84,6 +84,31 @@ def create_stack(self, stack_name, tags=None):
         data = self.get_responsebody(resp)
         return data
 
+    def update_stack(self, stack_name, stack_id, resp_code=202):
+        """
+        Request stack update and assert that the response matched the one
+        provided.
```

Comment:
Given that this is not a `@route` (and therefore its parameters are not documented by external documentation) would you mind adding `:param:` fields for `stack_name`, `stack_id`, and `resp_code`?  I'd really like Mimic to have a higher standard of internal API docs.